Docs/source/usage/parameters.rst

```diff
@@ -1892,6 +1892,21 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
         <diag_name>.adios2_operator.type = zfp
         <diag_name>.adios2_operator.parameters.precision = 3
 
+* ``<diag_name>.adios2_engine.type`` (``BP*``, ``SST``, ``SSC``, ``DataMan``) optional,
```

Comment:
I think these keys are case insensitive, thus lets keep them lowercase to avoid confusion with the spelling of the `openpmd_backend` parameter?
```suggestion
* ``<diag_name>.adios2_engine.type`` (``bp4``, ``sst``, ``ssc``, ``dataman``) optional,
```